_docs/server/configuration.md

```diff
@@ -56,17 +88,103 @@ The Lounge ships with 4 themes.
 [![Morning Theme][morning_thumb]][morning]
 [![Zenburn Theme][zenburn_thumb]][zenburn]
 
+[example]: /img/screenshots/example_css.png
+[example_thumb]: /img/screenshots/example_thumbnail.png (Example CSS)
+[crypto]: /img/screenshots/crypto_css.png
+[crypto_thumb]: /img/screenshots/crypto_thumbnail.png (Crypto CSS)
+[morning]: /img/screenshots/morning_css.png
+[morning_thumb]: /img/screenshots/morning_thumbnail.png (Morning CSS)
+[zenburn]: /img/screenshots/zenburn_css.png
+[zenburn_thumb]: /img/screenshots/zenburn_thumbnail.png (Zenburn CSS)
+
+## autoload
```

Comment:
This is going away in https://github.com/thelounge/lounge/pull/779, mind getting rid of it in this PR?